Fix lint errors (#27835)
TheOdinProject/curriculum: intermediate_html_css/intermediate_html_concepts/svgs.md

With vector graphics on the other hand, there's no grid. Instead, you have formulas for different shapes and lines. Since these are just formulas, it doesn't matter how large or small you want them to appear--they can scale to any size you want, and it will have no effect on the quality or the size of the file.

SVGs have another interesting aspect to them: they're defined using XML. XML (aka, "Extensible Markup Language") is an HTML-like syntax which is used for lots of things, from [APIs](https://en.wikipedia.org/wiki/API), to [RSS](https://en.wikipedia.org/wiki/RSS), to [spreadsheet and word editor software](https://en.wikipedia.org/wiki/Office_Open_XML).

The fact that SVG source-code is XML has a few key benefits.

First, it means that it is *human-readable*. If you were to open up a JPEG in a text editor, it would just look like gobbledygook. If you were to open up an SVG, however, it would look something like this:

> ```xml
> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
> 
>     <rect x=0 y=0 width=100 height=50 />
> 
>     <circle class="svg-circle" cx="50" cy="50" r="10"/>
> 
> </svg>
> 
> ```

Reviewer: ````suggestion
```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
  <rect x=0 y=0 width=100 height=50 />
  <circle class="svg-circle" cx="50" cy="50" r="10"/>
</svg>
```
````
Mind reverting the new line stuff and changing the code block language to `html`?
The linter won't complain if it's inside an HTML block. *Technically* `xml` is more correct but `html` is perfectly fine in context and this is the sole occurrence of an `xml` code block in the entire curriculum, so not worth adding to the TOP005 rule exceptions just yet.

### Knowledge check

The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.

- [What is the `xmlns` attribute?](#anatomy-of-an-svg)
- [What are some situations where you *wouldn't* want to use SVG?](#drawbacks)
- [What are the benefits of "inlining" your SVGs? What are the drawbacks?](#embedding-svgs)

### Additional resources

This section contains helpful links to related content. It isn't required, so consider it supplemental.

> - If you want to get started making your own or editing SVGs, you'll want some sort of visual editor like [Yann Armelin's SVG path editor](https://yqnn.github.io/svg-path-editor). It's great for tinkering around with simpler SVGs but not designed for complex graphics.

Reviewer: ```suggestion
- There are lots of great free SVG icon libraries, such as [Material icons](https://fonts.google.com/icons) and [Feather icons](https://feathericons.com/).
- If you want to get started making your own or editing SVGs, you'll want some sort of visual editor like [Yann Armelin's SVG path editor](https://yqnn.github.io/svg-path-editor). It's great for tinkering around with simpler SVGs but not designed for complex graphics.
```

You accidentally removed the first bullet when adjusting the second

Author: Ah, my bad! Let me just boot up my PC real quick to fix it.

### Knowledge check

The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.

- [What is the `xmlns` attribute?](#anatomy-of-an-svg)
- [What are some situations where you *wouldn't* want to use SVG?](#drawbacks)
- [What are the benefits of "inlining" your SVGs? What are the drawbacks?](#embedding-svgs)

### Additional resources

This section contains helpful links to related content. It isn't required, so consider it supplemental.

- There are lots of great free SVG icon libraries, such as [Material icons](https://fonts.google.com/icons) and [Feather icons](https://feathericons.com/).

> - If you want to get started making your own SVGs, you'll want some sort of visual editor.
>     1. If you wish to play around with creating or editing SVGs, check out [Yann Armelin's SVG path editor](https://yqnn.github.io/svg-path-editor). It's great for tinkering around with simpler SVGs but not designed for complex graphics.

Reviewer: ```suggestion
- If you want to get started making your own or editing SVGs, you'll want some sort of visual editor like [Yann Armelin's SVG path editor](https://yqnn.github.io/svg-path-editor). It's great for tinkering around with simpler SVGs but not designed for complex graphics.
```
In hindsight, because the AR section needed to be fixed to an unordered list instead, mind changing these two lines to be a single bullet like the above suggestion?
